ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel_correlation.ambr

```diff
@@ -3228,7 +3228,7 @@
   FROM
     (SELECT actor_id,
             funnel_actors.steps as steps,
-            arrayMap(x -> x.1, JSONExtractKeysAndValuesRaw(groups_0.group_properties_0)) as person_prop_keys,
+            JSONExtractKeys(groups_0.group_properties_0) as person_prop_keys,
```

Comment:
these snapshots should've had the `arrayJoin()` too no? 🤔

Reply:
Literally committed the change 30sec before the review :D 

https://github.com/PostHog/posthog/pull/14031/commits/890c7d699d529cba2b6f777f12686292e590fb1b

Reply:
Note that the scope changed from the original `JSONExtractKeys` change